.github/workflows/package.yml

```diff
@@ -0,0 +1,114 @@
+name: Package
+
+on:
+  push:
+    tags:
+      - v*.*.*
+
+jobs:
+
+  wheel-linux:
+    runs-on: ubuntu-latest
+    name: Build Linux wheels
+    strategy:
+      matrix:
+        include:
+        - python-version: 3.6
+          pyo3-python: /opt/python/cp36-cp36m/bin/python
+        - python-version: 3.7
+          pyo3-python: /opt/python/cp37-cp37m/bin/python
+        - python-version: 3.8
+          pyo3-python: /opt/python/cp38-cp38/bin/python
+        - python-version: 3.9
+          pyo3-python: /opt/python/cp39-cp39/bin/python
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v2
+      with:
+        submodules: true
+    - name: Build manylinux wheels
+      uses: addnab/docker-run-action@v2
+      with:
+        image: pypywheels/manylinux2010-pypy_x86_64
+        options: -v ${{ github.workspace }}:/io -e TERM=$TERM
+        shell: bash
+        run: |
+          ${{ matrix.pyo3-python }} -m pip install -U wheel auditwheel
+          ${{ matrix.pyo3-python }} -m pip wheel --no-deps -vv /io -w /tmp
+          for whl in /tmp/*.whl; do ${{ matrix.pyo3-python }} -m auditwheel repair $whl -w /io/dist; done
```

Comment:
It's being cautious, really, but there is an extra feature that can be useful: if `auditwheel repair` detects that your wheel is backward compatible with older platforms, it will add the relevant tag to the final wheel.

For instance, if you were to compile the wheel inside the `manylinux_2_24` docker image, at the end you'd get a file called `wrapt-1.12.2-cp39-cp39-linux_x86_64.whl`, which PyPI would reject on upload because it doesn't accept the `linux` platform tag. Since you know you were in `manylinux_2_24`, you could just rename it to `wrapt-1.12.2-cp39-cp39-manylinux_2_24_x86_64.whl` and upload it like that.

However, here's what happens if you run `auditwheel repair` instead:
```
INFO:auditwheel.main_repair:Repairing wrapt-1.12.2-cp39-cp39-linux_x86_64.whl
INFO:auditwheel.main_repair:Wheel is eligible for a higher priority tag. You requested manylinux_2_24_x86_64 but I have found this wheel is eligible for manylinux_2_5_x86_64.
INFO:auditwheel.wheeltools:Previous filename tags: linux_x86_64
INFO:auditwheel.wheeltools:New filename tags: manylinux_2_5_x86_64, manylinux1_x86_64, manylinux_2_24_x86_64
INFO:auditwheel.wheeltools:Previous WHEEL info tags: cp39-cp39-linux_x86_64
INFO:auditwheel.wheeltools:New WHEEL info tags: cp39-cp39-manylinux_2_5_x86_64, cp39-cp39-manylinux1_x86_64, cp39-cp39-manylinux_2_24_x86_64
```

It basically detected that the library was not linking against anything, and that it could even be tagged `manylinux1`, the oldest `manylinux` environment, meaning that virtually anybody could install that wheel now.